docs/userGuide/syntax/includes.md

```diff
@@ -61,6 +61,62 @@ The `<include>` mechanism can be used inside any MarkBind source file (even insi
 </span>
 </include>
 
+<panel header="Markdown not rendered when including a variable">
```

Comment:
The heading should let the reader decide quickly between expanding and reading the panel or skipping it. This one is hard to decide (is this a tip, advanced technique, caveat, etc.?), and the reader will have to expand the panel to figure out what it's about.

Reply:
Hmmm how about "If trying to include a variable with markdown" or something like that?